manifests/a/ActiveDirectoryPro/ADProToolkit/2.2.1/ActiveDirectoryPro.ADProToolkit.installer.yaml

```
@@ -0,0 +1,19 @@
+# Automatically updated by the winget bot at 2025/Jan/14
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ActiveDirectoryPro.ADProToolkit
+PackageVersion: 2.2.1
+InstallerLocale: en-US
+InstallerType: msi
+ProductCode: '{BE18DA8F-1FEF-45BB-9703-089F075520BB}'
+AppsAndFeaturesEntries:
+- DisplayName: AD Pro Toolkit
+  Publisher: ActiveDirectoryPro
+  ProductCode: '{BE18DA8F-1FEF-45BB-9703-089F075520BB}'
+  UpgradeCode: '{1A0C25E2-86F2-475B-B44E-7774AF3F3676}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://activedirectorypro.com/downloads/ADProToolkit.msi
+  InstallerSha256: 44CBD01772971C37813E421E3CFE0B7357DD817BAF34E3E8EA6E3D0CAF7077BF

```

```suggestion
  InstallerSha256: D8021D99576588A9B8EAB2D33C7A33BEBC2AC31388FF096FBABE1CD44E017A2F
```

(Automated response - build 943.)